salvo/src/lib/cmd_exec.py

```diff
@@ -17,7 +17,7 @@
     ])
 
 
-def run_command(cmd: str, parameters: CommandParameters) -> str:
+def run_command(cmd: str, parameters: CommandParameters, debug=True) -> str:
```

Comment:
The new parameter will break the unit tests, please revise their unit tests cases.

Reply:
@xu1zhou Adding a new parameter here will break the unit tests and you have to resolve many mock errors. Because this function is a lower level function used in many place, any small change may cause many unexpected issues, struggling with this may be a waste of time. 

Salvo is using `logging` package to manage logs, so you can use its functions such as [setlevel](https://docs.python.org/3/library/logging.html#logging.Logger.setLevel) to change the behaviors without adding any parameter.

Reply:
+1 on this comment. Adding a logging specific parameters makes the API more complicated and we do seem to have a viable alternative (setting the logging level).

Can we attempt a solution that will modify the logging level to compare? If you want, we can keep this PR around just in case we decide to go back to this and send a new PR that will set the logging level.